rosdep/python.yaml

```
@@ -6592,6 +6592,15 @@ python3-pycryptodome:
       packages: [pycryptodome]
   rhel: ['python%{python3_pkgversion}-pycryptodomex']
   ubuntu: [python3-pycryptodome]
+python3-pydantic:
+  arch: [python-pydantic]
+  debian: null
```

The package is available in the upcoming Bullseye: https://packages.debian.org/bullseye/python3-pydantic
```suggestion
  debian:
    bullseye: [python3-pydantic]
```

---

or to make it future proof:
```suggestion
  debian:
    '*': [python3-pydantic]
    buster: null
    jessie: null
    stretch: null
```

---

I added the latter.